### ISSUE TITLE
TypeError if a possible result isn't in the format [Image Dimensions] [Type]

### ISSUE THREAD

**Issue by GitThirteen** [l2studio/iqdb-api#3]
Good morning/afternoon/evening,

I'd like to report what is likely a bug due to IQDB (or rather, probably Gelbooru) rarely returning an `[Unrated]` without any image dimensions specified, which in turn makes line 186 in `api.js` fail with a TypeError, as dimensionTexts[1] ends up being `undefined`.

```js
function parseResultsPage($, $pageEl) {
  // ...
  const $dimensionEl = $($rows[$matchEl.length + 2]).find('td');
  const dimensionTexts = $dimensionEl.text().trim().split(' ');
  const dimension = dimensionTexts[0].replace(/[^\d]/g, '-').split('-');
  const width = parseInt(dimension[0]);
  const height = parseInt(dimension[1]);
  const type = dimensionTexts[1].substring(1, dimensionTexts[1].length - 1).toLowerCase(); // error occurs here
  const $similarityEl = $($rows[$matchEl.length + 3]).find('td');
  const similarityTexts = $similarityEl.text().trim().split(' ');
  // ...
```

Edit: I just cross-checked with the code in the repository, it corresponds to **line 214** in `api.ts`.
```ts
// dimensionTexts[1] is undefined, therefore .substring() causes a TypeError
const type = dimensionTexts[1].substring(1, dimensionTexts[1].length - 1).toLowerCase() 
```

As seen here, the `dimensionTexts` aren't in the format `WidthXHeight [Type]`, but just `[Type]`:
![error](https://user-images.githubusercontent.com/51542804/229319832-87390a9d-0cd1-4b8f-9fc3-824c3ced207d.png)
while the code assumes that the result should look like this: 
![expected](https://user-images.githubusercontent.com/51542804/229319896-9d33a757-83fb-491b-a5e6-9f98bbcc46f8.png)

Amazing library otherwise! Cheers.

**Comment by lgou2w** [comment]
Thank you very much for your feedback. This bug has been confirmed!

**Comment by lgou2w** [comment]
This bug was fixed in [v0.2.1](https://github.com/l2studio/iqdb-api/releases/tag/0.2.1)
